bodhi/client/__init__.py

```diff
@@ -688,18 +690,21 @@ def download(url, **kwargs):
     Download the builds for an update.
 
     Args:
-        staging (bool): Whether to use the staging server or not.
-        arch (unicode): Requested architecture of packages to download.
-                        "all" will retrieve packages from all architectures.
-        url (unicode): The URL of a Bodhi server to create the update on. Ignored if staging is
-                       True.
-        kwargs (dict): Other keyword arguments passed to us by click.
+        staging (bool):   Whether to use the staging server or not.
+        arch (unicode):   Requested architecture of packages to download.
+                          "all" will retrieve packages from all architectures.
+        debuginfo (bool): Whether to include debuginfo packages.
+        url (unicode):    The URL of a Bodhi server to create the update on. Ignored if staging is
+                          True.
+        kwargs (dict):    Other keyword arguments passed to us by click.
     """
     client = bindings.BodhiClient(base_url=url, staging=kwargs['staging'])
     requested_arch = kwargs['arch']
+    debuginfo = bool(kwargs['debuginfo'])
```

Comment:
I think ```click``` will make this a ```bool``` for us, since it uses the ```is_flag=True``` argument above.

Reply:
I just wanted to be sure we took a copy not a reference before we delete it below.